lib/renderer.js

```diff
@@ -1265,11 +1265,13 @@
             canvasDimensionsScale = 1,
             settingsPromise,
             resultPromise,
+            isVisible = layer && layer.visible,
```

Comment:
`isVisible` will be false if there is no layer (ie, for layerComp and maybe whole-doc).  I'm not sure how critical this distinction is, but strictly speaking the logic should probably be changed to explicitly set a variable like `isInvisible` ...

Also need to add this condition to the comment/docs above.